src/services/PrivateKeyService.js

```diff
@@ -66,6 +80,28 @@ class PrivateKeyService {
       throw new EthereumAddressError(err)
     }
   }
+
+  /**
+   * It gets the private key from mnemonic
+   *
+   * @param {String} mnemonic The mnemonic.
+   *
+   * @throws {InvalidMnemonicError} If the menomic is not valid.
+   */
+  getPrivateKeyFromMnemonic(mnemonic) {
+    try {
+      const validMnemonic = this.privateKeyValidator.validateMenmonic(mnemonic)
+      const hdwallet = hdkey.fromMasterSeed(bip39.mnemonicToSeed(validMnemonic))
+      const wallet = hdwallet.derivePath(`${this.hdPath}0`).getWallet()
```

Comment:
I would use BIP-44:
A) explicit that the main account is returned
B) give the option to ask for a specific `account_index`

Reply:
I have added hdPath in PrivateKeyService constructor.